adapter/adapter.go

```diff
@@ -64,7 +67,7 @@ func NewAdapter(
 		things:    make(map[string]Thing),
 		factory:   factory,
 		state:     state,
-		publisher: NewPublisher(mqtt, resourceName, resourceAddress),
+		publisher: NewPublisher(eventManager, mqtt, resourceName, resourceAddress),
```

Comment:
ctor -> (mqtt, eventManager, ...)
NewPublisher -> (eventManager, mqtt, ...)

Reply:
@viktor-viktor Please fix this in your PR.